pyinfoblox/__init__.py

```diff
@@ -68,19 +77,45 @@ def __getattr__(self, attr):
         """
         Dynamically create a new Infoblox object class, e.g. 'network'
 
+        Args:
+            attr (str): The WAPI Object name that's requested
+
+        Returns:
+            InfobloxWAPIObject (obj): The Infoblox Object session output/payload
+
         """
-        # Special case for 'record' objects.
+        # Special case for root objects, with subobjects under them.
         #
-        # The Infoblox 'record' objects are in the following form:
+        # Infoblox objects with children are in the following form:
         #
         #     'record:<objtype>'
         #
         # For example A records in Infoblox are 'record:a' objects.
         #
-        # In order to use an Infoblox 'record' object replace the
+        # In order to use an Infoblox child-object replace the
         # colon character with underscore in your call, e.g. 'record_a'
-        if 'record' in attr:
-            attr = attr.replace('_', ':', 1)
+
+        # all root objects that have children, according to WAPI v2.9
+        root_objects = ['certificate','ciscoise','ddns','dhcp','discovery',\
```

Comment:
No need for backslashes at the end of each line.

Reply:
Fixed.